### DISCUSSION TITLE
How to opt out of dev versions?

### DISCUSSION THREAD

**Discussion by totalitarian** [greghesp/ha-bambulab#706] [Q&A]
Love how active the development is here but i'm getting daily update reminders in HACs. Is there a way to only get stable updates?

**Comment by AdrianGarside** [comment]
Those weren't dev versions. They were critical fixes to try to resolve blocking authentication issues with Bambu's rollout of cloudflare. HACS has a setting to not show you prerelease versions that you would have had to turn on to see true dev versions.
